translations/en_GB/atrium-translations-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionPathAssertion.kt

```diff
@@ -8,6 +8,7 @@ package ch.tutteli.atrium.translations
 import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
 
 enum class DescriptionPathAssertion(override val value: String) : StringBasedTranslatable {
+    ABSOLUTE_PATH("absolute path"),
```

Comment:
```suggestion
    ABSOLUTE_PATH("an absolute path"),
```
If I read the code correct, the failure message will currently read “is relative path”, which is missing an article.